### PR TITLE
Folder Refactor

### PR DESCRIPTION
**Pull request by hahnbeelee**
### Summary
- Added a `containers` folder where the main "pages" lie (Login, 404, Dashboard, Settings (?))
- Organized components into folders
   - All components related to Requirements/SemesterView/BottomBar in one folder
   - Components that are re-used in multiple places are kept at the highest level `Components` folder ( e.g. `Course.vue` and `DropdownArrow.vue`)

### Test Plan
- Make sure `npm run serve` works
- Look through the file structure and see if it makes sense
  - specifically, see if there are some components in sub-folders that should be at the highest level.

### Notes
We are holding off on merging this till winter break to reduce merge conflicts while people are finishing up their final PRs for the semester.

### CODE REVIEW

File: src/components/Navigation.vue [hunk 1]
```diff
@@ -1,9 +0,0 @@
-<template>
```
Comment (hahnbeelee) [inline src/components/Navigation.vue:1]: this didn't seem like it was being used anywhere and my GitLens extension was telling me that it was added on `first commit` so I'm assuming this is outdated lol

**Review by SamChou19815 — APPROVED** [review]
Actually looks good to go. git can detect changed files so merge conflicts won't be too bad.

**Comment by hahnbeelee** [comment]
@willespencer thoughts on just merging this rn?

**Review by willespencer — CHANGES_REQUESTED** [review]
Thanks for doing this Hahnbee! I'm trusting Sam on this one so I do think we can merge it in, just make sure you mention that this happened in cp-dev after this gets finished. I also have a couple suggested changes haha.

- Can you move the 4 onboarding components and 1 scss file to their own separate folder as well?
- Can you delete Settings.vue altogether instead of moving it? It is a remnant from our starter code as well. You probably have to remove it from the Router too.

**Comment by hahnbeelee** [comment]
> Thanks for doing this Hahnbee! I'm trusting Sam on this one so I do think we can merge it in, just make sure you mention that this happened in cp-dev after this gets finished. I also have a couple suggested changes haha.
> 
> * Can you move the 4 onboarding components and 1 scss file to their own separate folder as well?
> * Can you delete Settings.vue altogether instead of moving it? It is a remnant from our starter code as well. You probably have to remove it from the Router too.

Done!

**Review by willespencer — APPROVED** [review]
Looks good! Talked over slack but let's just wait till #159 is merged in to be extra safe!

**Pull request merged**